app/src/main/java/com/example/the_dagger/learnit/activity/LoginActivity.java

```diff
@@ -133,6 +132,44 @@ public void onErrorResponse(VolleyError error) {
         });
     }
 
+
```

Comment:
Unnecessary white lines. Due to lack of time we might have not followed coding guidelines properly but you should follow it before sending PR